Makefile

```diff
@@ -52,7 +52,9 @@ fresh: ## default target - sets up a k8s cluster with images ready for deploymen
 		operator-build operator-push \
 		bundle bundle-build \
 		bundle-push \
-		IMG_BASE=localhost:5001 VERSION=0.0.0-dev ;\
+		IMG_BASE=localhost:5001 VERSION=0.0.0-dev \
+		KEPLER_IMG=quay.io/sustainable_computing_io/kepler:$(KEPLER_VERSION) \
+		KEPLER_IMG_LIBBPF=quay.io/sustainable_computing_io/kepler:$(KEPLER_VERSION_LIBBPF)  ;\
```

Comment:
I think we should fix how `KEPLER_IMAGE`(s) are initialised. How about we create a `KEPLER_IMG_BASE` which always points to `quay.io/...` by default and use `IMG_BASE` only for OPERATOR related images ?